scheduler/scheduler-examples/src/main/java/org/ow2/proactive/scheduler/examples/SchedulerExamplesUtils.java

```diff
@@ -148,4 +148,23 @@ protected static void eraseProgressBar() {
         }
         return filesRelativePathName;
     }
+
+    /**
+     * Creates a directory if it does not exist
+     * @param file
+     */
+    protected static void createDirIfNotExists(File file) {
```

Comment:
now i am thinking when do we need both static and protected thing. But it seems there could be use cases. Could you share with me your motivation to use "protected static"?

Reply:
-"protected" because it is used by classes of the same "examples" package
-"static" because no need to instantiate the class

Reply:
ok.